site/src/templates/standard-article.js

```diff
@@ -161,7 +163,19 @@ export default class Article extends React.Component {
     }
     return false
   }
-
+  hasCCNote(post) {
+    if (!this.IS_CC_SITE) {
```

Comment:
Does this not fall foul of the false vs "false" truthiness problem?

Reply:
fixed